mozilla-services/socorro
# fix bug 1483097: fix product home

This fixes the product home (and the nav header) to work with products that may not have information in the `product_versions` table.

To test this, do:

1. `make setup`
2. `make run`
3. open up your browser and go to `http://localhost:8000`

You'll see a bunch of products which have no information in `product_versions`. The product home page won't show any Top Crasher links. If you process some crashes thus adding crash report data, there will then be Top Crasher links.

Hooray!
